ext/shopify-extensions/extconf.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "./shopify_extensions"
+
+File.write("Makefile", <<~MAKEFILE)
+  .PHONY: clean
+
+  clean: ;
+
+  install: ;
+MAKEFILE
+
+begin
+  ShopifyExtensions.install(
```

Comment:
>  Is it because we don't want to include the binaries of architectures other than ours?
That is correct. We've considered this option but dismissed it due to binary size. The binaries are quite largue as each one includes the Go runtime: ~ 10 MB per binary (5 MB compressed)

> Have we considered Nokogiri's approach where multiple native gems are generated containing the right binary for each architecture?
We have not. I wasn't aware that this option exists. From the link you provided, it's not immediately evident to me how to actually package and publish OS specific versions of Ruby gems. Do you have any experience with this yourself and could elaborate a bit?

Reply:
The approach we took follows the one that ESBuild uses to obtain its binary when you install the corresponding node package. Can you expand a bit on the concerns you have with this approach?

Reply:
@pepibumur, do you see this as a blocker or something that we could improve incrementally. Since we're not releasing to partners before my intermission, I'd be ok to wait with integrating that, but it makes testing this a little more manual.